tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/migration/MigrationActionProvider.java

```diff
@@ -88,6 +90,17 @@ public void fillContextMenu( IMenuManager menu )
 
             final TaskProblem problem = (TaskProblem) selection;
 
+            if( problem.isResolved() )
```

Comment:
Add this logic to each of the actions separately in the "selectedChanged" method.